Makefile

```diff
@@ -3,6 +3,13 @@ format:
 	cargo +stable fmt --all
 .PHONY: format
 
+build: setup-git
+	cargo +stable build
+.PHONY: build
+
+release: setup-git
+	@cargo +stable build --release
```

Comment:
do you need the `@` here?

Reply:
Removed it. I copied this from relay so it was left behind.